docs/AddNewOp.md

```diff
@@ -44,33 +44,33 @@ Once the criteria of proposing new operator/function has been satisfied, you wil
     4. Write the mathematic formula or a pseudocode in the description. The core algorithm needs to be very clear.
 2. Write a reference implementation in Python, this reference implementation should cover all the expected behavior of the operator. Only in extremely rare case, we will waive this requirement.
 3. Operator version: check out our
-[versioning doc](https://github.com/onnx/onnx/blob/master/docs/Versioning.md#operator-versioning)
+[versioning doc](/docs/Versioning.md#operator-versioning)
 4. Write unit test, that cover main usage and corner cases.
     1. The testing examples will be extracted to the doc.
     2. We also generate binary data for it.
-    3. Example: https://github.com/onnx/onnx/blob/master/onnx/backend/test/case/node/abs.py
-5. Add at least one automatic upgrade test for your operator in https://github.com/onnx/onnx/blob/master/onnx/test/automatic_upgrade_test.py using `_test_op_upgrade`. These tests create a given operator at a given opset version (usually the version the operator was introduced in) and test that the version converter is able to convert them to the highest available version. So for a new operator `_test_op_upgrade` will not test anything, but as soon as the operator gets updated in a future opset the test will autoamtically become nontrivial.
+    3. Example: [onnx/backend/test/case/node/abs.py](/onnx/backend/test/case/node/abs.py)
+5. Add at least one automatic upgrade test for your operator in [onnx/test/automatic_upgrade_test.py](/onnx/test/automatic_upgrade_test.py) using `_test_op_upgrade`. These tests create a given operator at a given opset version (usually the version the operator was introduced in) and test that the version converter is able to convert them to the highest available version. So for a new operator `_test_op_upgrade` will not test anything, but as soon as the operator gets updated in a future opset the test will autoamtically become nontrivial.
 6. Update the documentation and generate the test data.
-    1. Running [the script](https://github.com/onnx/onnx/blob/master/tools/update_doc.sh)
+    1. Running [the script](/tools/update_doc.sh)
 to update the doc and generate the test data.
 7. Shape Inference function
     1. Please provide a shape inference function in cases where it is meaningful and applicable.
     2. In cases where shape inference is not possible, it must have logic to perform
 rank inference at the very least (adding right amount of dimensions to the output shape)
-    3. Shape inference functions must be accompanied by unit tests (https://github.com/onnx/onnx/blob/master/onnx/test/shape_inference_test.py).
-    4. You can refer to the shape inference function for the `TopK` operator while implementing your own function (https://github.com/onnx/onnx/blob/master/onnx/defs/math/defs.cc#L943)
```

Comment:
Please note that I removed #L943 here since defs.cc keeps changing. People should be able to figure out where TopK is from their own.